app/wp-content/themes/kstg/library/scss/components/_home-page.scss

```diff
@@ -0,0 +1,157 @@
+.homepage-wrapper {
+  background-color: $light-grey;
+  section {
+    padding: 0 30px 100px 30px;
+    max-width: 1220px;
+    margin: 0 auto;
+    .gallery-container {
+      margin: 60px 25px 50px -5px;
+      display: inline-block;
+      div {
+        padding-left: 30px;
+      }
+      .gallery-corporate, .gallery-events, .gallery-travel, .gallery-wedding {
```

Comment:
This stuff is all subjective, but a good best practice is to put everything on its own line. Example:

```css
.gallery-corporate,
.gallery-events,
.gallery-travel,
.gallery-wedding {
  /* properties */
}